README.md

```diff
@@ -11,8 +11,8 @@ Image is debian-based and includes default command line tools and all [component
 
 ## Supported tags and respective Dockerfile links
 
-> * ```google/cloud-sdk:latest```, ```google/cloud-sdk:VERSION```: (large image with additional components pre-installed, Debian-based)
-> * ```google/cloud-sdk:slim```,  ```google/cloud-sdk:VERSION-slim```: (smaller image with no components pre-installed, Debian-based)
+* ```:latest```, ```:VERSION```: (large image with additional components pre-installed, Debian-based)
```

Comment:
ok, i was just looking at some other examples on dockerhub (eg (alpine)[https://hub.docker.com/_/alpine/] but since its on github too, its easier with the full name)